docs/index.rst

```diff
@@ -793,8 +793,10 @@ Confluent REST Proxy
 --------------------
 
 The `Confluent REST Proxy <https://docs.confluent.io/current/kafka-rest/docs/index.html>`__  is running for optional client access.
+This demo showcases |crest| in two modes: (1) running as a standalone service and (2) embedded on the |ak| brokers.
+While in embedded mode |crest| listens for requests on port 8092 providing only the :ref:`rest-proxy-v3` are supported this time.
```

Comment:
There are two ERP instances, one on broker1 and one on broker2.  In Docker they shouldn't both be listening on port 8092.  Can it be differentiated?

Reply:
MDS and KafkaRest share the same HTTP server so they have the same listeners. I can add the full address for both